css_tasks/task6/main.css

```diff
@@ -0,0 +1,453 @@
+  color: var(--mainColor);
+  font-family: var(--mainFont);
+  font-size: var(--biggerFontSize);
+}
+.content__blog--paragraph {
+  color: var(--mainColor);
+  font-family: var(--mainFont);
+  font-size: var(--mainFontSize);
+}
+.content__blog-button {
+  grid-column: span 2;
+  width: 10.25rem;
+  height: 3.75rem;
+  margin: 0 auto;
+  color: var(--mainColor);
+  font-family: var(--mainFont);
+  font-size: 1.25rem;
+  border: 1px solid var(--mainColor);
+  background: none;
+  cursor: pointer;
+}
+/************************* 
+*                        *
+*       NEWSLETTER       *
+*                        *
+*************************/
+
+.newsletter {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  width: 100%;
+  height: 17.9375rem;
+  background-color: var(--backgroundColor);
+}
+.newsletter__title {
+  padding-top: 0.625rem;
+  color: #333;
+  font-family: var(--mainFont);
+  font-size: var(--biggerFontSize);
+  letter-spacing: 0.8px;
+}
+.newsletter__form {
+  display: flex;
+  align-items: center;
+  height: var(--biggerFontSize);
+  width: 25rem;
+  padding-bottom: 0.875rem;
+  border-bottom: 1px solid var(--logoColor);
+}
+.newsletter__input {
+  width: 95%;
+  color: var(--mainColor);
+  font-family: var(--mainFont);
+  font-size: var(--mainFontSize);
+  border: none;
+  background: none;
+  outline: 0;
+}
+.newsletter__input::placeholder,
+.newsletter__input:focus {
+  color: var(--mainColor);
+  font-family: var(--mainFont);
+  font-size: var(--mainFontSize);
+}
+.newsletter__button {
+  width: 4%;
+  background: none;
+  border: none;
+  cursor: pointer;
+}
+.newsletter__button-icon {
+  color: var(--mainColor);
+}
+/************************* 
+*                        *
+*        FOOTER          *
+*                        *
+*************************/
+.footer {
+  height: 6.0625rem;
+  background-color: var(--backgroundColor);
+}
+.footer__main-container {
+  display: flex;
+  width: 55.8125rem;
+  height: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+.footer__element-text {
+  padding-right: 1.0625rem;
+  font-family: var(--mainFont);
+  font-size: var(--mainFontSize);
+}
+.footer__element:last-child .footer__element-text {
+  padding-right: 0;
+}
+.footer__element img {
+  height: 0.8125rem;
+  padding-left: 1.0625rem;
+  cursor: pointer;
+}
+/************************* 
+*                        *
+*      MEDIA QUERIES     *
+*                        *
+*************************/
+
+@media screen and (min-width: 992px) and (max-width: 1199px) {
+  .container {
+    max-width: 60rem;
+  }
+  .content__blog,
+  .content__blog-secondary,
+  .content__main-text,
+  .footer__main-container {
+    width: 48rem;
+  }
+}
+@media screen and (min-width: 768px) and (max-width: 991.98px) {
+  .header__logo {
+    margin-right: 5rem;
+  }
+  .content__blog,
+  .content__blog-secondary,
+  .content__main-text,
+  .footer__main-container {
+    width: 36rem;
```

Comment:
works same with 100%(remove as default is 100%), probably you can change it just to % in all other places and everything will look fine(content will fit to 80% of content width), and responsive without media queries, web is responsive by default just need to add some magic :)